src/app/app.component.ts

```diff
@@ -730,9 +734,111 @@ export class AppComponent implements OnInit {
         }
         this.comics[comicIndex].styles.background = collectionColor.backgroundColor;
         this.comics[comicIndex].styles.color = collectionColor.textColor;
+
+        //  while we are here store the comics in reading order
+        this.comicsInReadingOrder.push(_.cloneDeep(this.comics[comicIndex]));
       });
     });
 
+    //  reposition the reading order comics horizontally
+    let horizontalReadingOrderPosition = 0;
+    _.each(this.comicsInReadingOrder, (comic) => {
+      horizontalReadingOrderPosition += VISUAL_BLOCK_SIZE;
+      comic.containerStyles['left.px'] = horizontalReadingOrderPosition;
+
+      /*
+       * Look up the volume and series for this comic and
+       * throw errors for obvious problems before proceeding.
+       */
+      const currentSeriesVolume = this.seriesVolumes[_.findKey(this.seriesVolumes, { id: comic.seriesVolumeId })];
+      if (!currentSeriesVolume) {
+        throw new Error(comic.seriesVolumeId + ' not found');
+      }
+      // this is a copy of the other positioning
+      /*
+       * Vertical positioning ensures that each seriesVolume gets
+       * its own row on the page. The exception is if a seriesVolume
+       * has not had any new issues for a whole page width, then we
+       * allow the next seriesVolume that is looking for a row, to slot in.
+       *
+       * This is a two-step process.
+       * First, in this block, we either use the current position for the
+       * seriesVolume or we use the globalVerticalPositionCounter to go onto
+       * the last row.
+       * Step two documented below.
+       */
+
+       // TO-DO: Paulene really needs to understand what's happening here
+       // so I added this line because I couldn't get things to line up.  Unsure what it should be reset to.
+      this.globalVerticalPositionCounter = 1;
+      // this if statement checks that the vertical position attribute of the currentSeriesVolume is not undefined
+      // I'm wondering when it would be undefined.
+      if (typeof currentSeriesVolume.verticalPosition !== 'undefined') {
+        // here we set the vertical position for this comic.
+        // this must be a counting integer since we multiply it by the visual block size
+        comic.containerStyles['top.px'] = currentSeriesVolume.verticalPosition * VISUAL_BLOCK_SIZE;
+      } else {
+        // if the vertical position is undefined (does this mean that it's not set?)
+        // we set the vertical position to the global position.
+        // so I'm assuming this holds the "next" position?
+        currentSeriesVolume.verticalPosition = this.globalVerticalPositionCounter;
+        // so here I'm wondering why we set the currentSeriesVolume.verticalPosition and what that does
+        // since here we set the top to the global counter * the constant block size
```

Comment:
On line `784` you can see we set the `verticalPosition` to the global counter, so doing:

`comic.containerStyles['top.px'] = this.globalVerticalPositionCounter * VISUAL_BLOCK_SIZE;`

is equivalent to doing:

`comic.containerStyles['top.px'] = currentSeriesVolume.verticalPosition * VISUAL_BLOCK_SIZE;`

However that should be done more readably, do you mind if I do a small commit to this branch for that?

Reply:
Of course not

Reply:
Okay I'm understanding this now

Reply:
It seems like the only reason we use the currentSeriesVolume.verticalPosition is so that we know whether it's undefined or set, because otherwise we could just use the global counter.

Reply:
Well if we used the global counter all the time we would be incrementing a lot - the reason for `currentSeriesVolume.verticalPosition` is to make all issues in a seriesVolume chunk sit on the same row